doc/rst/platforms/udp.rst

```diff
@@ -153,9 +153,10 @@ I get xSocket errors when using a system with multiple IP addresses
  *** FATAL ERROR: Got an xSocket while spawning slave process: connect()
  failed while creating a connect socket (111:Connection refused)
 
-You need to set ``GASNET_MASTERIP`` and possibly ``GASNET_WORKERIP``.
-Please refer to:
+You need to set ``CHPL_LAUNCH_MASTERIP`` (or ``GASNET_MASTERIP``), and possibly
+``GASNET_WORKERIP``.  Please refer to:
```

Comment:
Might be nicer to provide chapel env vars for master and worker IP, instead of mixing between chapel and gasnet vars.

Reply:
Sure.  I didn't at the time because I don't think we'll need the worker IP for the case that triggered the need for the other one.